example/index.html

```diff
@@ -1,36 +1,36 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>NoSleep.js - Simple Test Page</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-  </head>
-  <body>
-    <h1>NoSleep Test Page</h1>
-    <script src="../dist/NoSleep.min.js"></script>
+<head>
+  <title>NoSleep.js - Simple Test Page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+<h1>NoSleep Test Page</h1>
+<script src="../dist/NoSleep.min.js"></script>
```

Comment:
This diff seems to originate due to the use of different format settings from the repo's owner. I suggest reverting those changes because the owner may not like them